packages/documentation/copy/pt/reference/Namespaces and Modules.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Namespaces e Módulos
+layout: docs
+permalink: /pt/docs/handbook/namespaces-and-modules.html
+oneline: Como organizar o código em TypeScript através de módulos ou namespaces
+translatable: true
+---
+
+Este post descreve as várias formas de organizar seu código usando módulos e namespaces no TypeScript.
+Abordaremos também alguns tópicos avançados sobre como usar namespaces e módulos, e algumas armadilhas comuns ao usá-los no TypeScrypt.
+
+Veja a documentação de [Módulos](/docs/handbook/modules.html) para mais informações sobre os módulos do ES.
+Veja a documentação de [Namespaces](/docs/handbook/namespaces.html) para mais informações sobre os namespaces do Typescript. 
```

Comment:
Same as above goes here but the `namespaces.html` is available in Portuguese so you can freely change it.

```suggestion
Veja a documentação de [Namespaces](/pt/docs/handbook/namespaces.html) para mais informações sobre os namespaces do Typescript. 
```

Reply:
Thank you for your help! I'll fix this one asap.